intake/catalog/base.py

```diff
@@ -191,6 +191,10 @@ def walk(self, sofar=None, prefix=None, depth=2):
             out[n] = item
         return out
 
+    def items(self):
+        """Get an iterator over (key, value) tuples for the catalog entries."""
+        return self._get_entries().items()
```

Comment:
@danielballan , might this be a problem for catalogs where the entries are not a simple dict?

Reply:
Thanks for thinking of that, but this should be OK. The dict-like `Entries` class used by `RemoteCatalog` implements `items()`, as do the special dict-like classes I wrote for intake-bluesky. In fact I use `cat._get_entries().items()` a couple places in intake-bluesky as a workaround and was considering asking for `cat.items()` to be added. Thanks @ian-r-rose!

Added https://github.com/intake/intake/pull/318 to document which parts of the dict interfaces intake expects custom entries containers to implement.